auth4genai/snippets/get-started/langchain-fastapi-py/call-others-api.mdx

```diff
@@ -340,40 +443,16 @@ export function ChatWindow(props: {
 
 ### Test your application
 
-To test the application, start the FastAPI backend, LangGraph server, and the frontend:
 
-1. In a new terminal, start the FastAPI backend:
+Now when step-up authorization is required, the user will see a prompt in the chat window asking them to authorize.
 
-```bash wrap lines
-cd backend
-source .venv/bin/activate
-fastapi dev app/main.py
-```
-
-2. In another terminal, start the LangGraph server:
-
-```bash wrap lines
-cd backend
-source .venv/bin/activate
-uv pip install -U langgraph-api
-langgraph dev --port 54367 --allow-blocking
-```
-<Note>
-  This will open the LangGraph Studio in a new tab. You can close it as we won't
-  require it for testing the application.
-</Note>
-
-3. In another terminal, start the frontend:
-
-```bash wrap lines
-cd frontend
-cp .env.example .env # Copy the `.env.example` file to `.env`.
-npm install
-npm run dev
-```
+### Test your application
```

Comment:
Currently attempting to reflect this style w/ a new quickstart here 😸 . Looking good!

Just noting "Test your application" used twice here ^^